buildchain/buildchain/targets/package.py

```diff
@@ -319,3 +320,114 @@ def _file_from_url(url: str) -> str:
     """Get filename from a URL."""
     path = urllib.parse.urlparse(url).path
     return urllib.parse.unquote(os.path.basename(path))
+
+
+class DEBPackage(Package):
+    """A DEB software package for Ubuntu 18.04."""
+
+    ARCH = 'amd64'
+
+    def __init__(
+        self,
+        basename: str,
+        name: str,
+        version: str,
+        sources: Path,
+        build_id: int,
+        builder: image.ContainerImage,
+        **kwargs: Any
+    ):
+        super().__init__(
+            basename, name, version, build_id, builder,
+            constants.PKG_DEB_ROOT, **kwargs
+         )
+        self._sources = sources
+
+    sources = property(operator.attrgetter('_sources'))
```

Comment:
I think that with a little work the `sources` management can me mutualized into the `Package` class. This can be an improvement ticket I guess.

Reply:
I create a ticket, you can follow the issue REF #1623